<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recognizer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>0</y>
     <width>802</width>
     <height>600</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Snoudin/.designer/backup/Фон основной.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="surmane">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="patronym">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="birth">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="born_in">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Место рождения</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gender">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Пол</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="code">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Код подразделения</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="receiving_date">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>440</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Дата выдачи</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="receiving_place">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Паспорт выдан</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>Загрузить фото</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Загрузить фото</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Snoudin/.designer/backup/Загрузка 2.png</normaloff>C:/Users/Snoudin/.designer/backup/Загрузка 2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="surname_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="patronym_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="born_in_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>280</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="birth_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="gender_ans">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>140</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="receive_date_ans">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>440</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>256</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>Обработать фото</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Обработать</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Snoudin/.designer/backup/analyze.png</normaloff>C:/Users/Snoudin/.designer/backup/analyze.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="code_ans">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>400</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="received_in_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>479</y>
     <width>280</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Номер</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="series">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Серия</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="number_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="series_ans">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fulbo Argenta</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>256</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>Загрузить фото</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранить правки</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Snoudin/.designer/backup/Загрузка 2.png</normaloff>C:/Users/Snoudin/.designer/backup/Загрузка 2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>birth_ans</zorder>
  <zorder>patronym</zorder>
  <zorder>gender</zorder>
  <zorder>surmane</zorder>
  <zorder>patronym_ans</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>receiving_place</zorder>
  <zorder>code</zorder>
  <zorder>name_ans</zorder>
  <zorder>born_in_ans</zorder>
  <zorder>birth</zorder>
  <zorder>born_in</zorder>
  <zorder>gender_ans</zorder>
  <zorder>receive_date_ans</zorder>
  <zorder>surname_ans</zorder>
  <zorder>name</zorder>
  <zorder>pushButton</zorder>
  <zorder>receiving_date</zorder>
  <zorder>code_ans</zorder>
  <zorder>received_in_ans</zorder>
  <zorder>number</zorder>
  <zorder>series</zorder>
  <zorder>number_ans</zorder>
  <zorder>series_ans</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
